qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -1510,7 +1510,7 @@ QString ToQStr(const ResourceUsage usage, const GraphicsAPI apitype)
       case ResourceUsage::GS_Constants: return lit("GS - Constant Buffer");
       case ResourceUsage::HS_Constants: return lit("TCS - Constant Buffer");
       case ResourceUsage::DS_Constants: return lit("TES - Constant Buffer");
-      case ResourceUsage::PS_Constants: return lit("FS - Constant Buffer");
+      case ResourceUsage::PS_Constants: return lit("PS - Constant Buffer");
```

Comment:
I see what you mean about the tessellation shaders, but regarding mesh shaders: do you mean that TS_Resource for instance for D3D should be AS (amplification shader) instead of TS (task shader)? Apologies but I'm not too familiar with the terminology outside vertex/pixel/compute.

Reply:
Yes exactly, it should be AS in D3D mode.